packages/keyring-internal-snap-client/package.json

```diff
@@ -45,19 +45,18 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
-    "@metamask/keyring-api": "workspace:^",
     "@metamask/keyring-snap-client": "workspace:^",
-    "@metamask/keyring-utils": "workspace:^",
-    "@metamask/snaps-controllers": "^9.10.0",
```

Comment:
We actually only need types from `snaps-*` packages, so we moved everything into `devDependencies`.